falcon-taz/src/main/java/pt/haslab/taz/TraceProcessor.java

```diff
@@ -60,7 +60,7 @@ The message id is increased whenever a pair (snd,rcv) is complete */
     public Map<SocketEvent, List<Event>> handlerEvents;
 
     /* set indicating whether a given thread's trace has message handlers */
-    private HashSet hasHandlers;
+    public HashSet<String> hasHandlers;
```

Comment:
Why not keeping it private?

Reply:
Fixed in commit a0b1e252fbe76fdb32b5a87bffef378b99dc2e79